app/commands/promoted_click_accounter.rb

```diff
@@ -0,0 +1,11 @@
+class PromotedClickAccounter < PowerTypes::Command.new(:product_action, :store, :organization)
+  include Ledgerizer::Execution::Dsl
+
+  def perform
+    amount = Money.from_amount(ENV['CPC'])
+    execute_promoted_click_entry(tenant: @organization, document: @product_action, date: Date.today) do
```

Comment:
Do not use Date.today without zone. Use Time.zone.today instead.<br>Line is too long. [103/100]